unlock-app/src/components/interface/locks/Create/elements/CreateLockForm.tsx

```diff
@@ -60,18 +60,23 @@ export const networkDescription = (network: number) => {
           Mainnet.
         </p>
       )}
-      {faucet && (
-        <>
-          {' '}
-          <br />
-          Need some {nativeCurrency.name} to pay for gas?{' '}
-          <Link className="underline" href={faucet} target="_blank">
-            Try this faucet
-          </Link>
-          .
-        </>
+      {faucets && (
+        <div className="mt-1">
+          Need some {nativeCurrency.name} to pay for gas? Try one of these
+          {faucets.length > 1 ? ' faucets' : ' faucet'}:{' '}
```

Comment:
This is going to read "Try one of these faucet" which is should rather be "Try his faucet".